test/integration/testpaths/testpath_multiple_snapshot.py

```
@@ -0,0 +1,255 @@
+
+            self.pools = StoragePool.list(self.apiclient, zoneid=self.zone.id)
+            if len(list(storagePool for storagePool in self.pools
+                        if storagePool.scope == "ZONE")) < 1:
+                self.skipTest("There must be at least one zone wide\
+                storage pools available in the setup")
+
+            if len(list(storagePool for storagePool in self.pools
+                        if storagePool.scope == "CLUSTER")) < 2:
+                self.skipTest("There must be at atleast two cluster wide\
+                storage pools available in the setup")
+        except Exception as e:
+            self.skipTest(e)
+
+        # Adding tags to Storage Pools
+        zone_no = 1
+        self.debug("Storage Pools: %s" % self.pools)
+        self.zone_pool = []
+        for storagePool in self.pools:
+            if storagePool.scope == "ZONE":
+                StoragePool.update(
+                    self.apiclient,
+                    id=storagePool.id,
+                    tags=[ZONETAG1[:-1] + repr(zone_no)])
+                self.zone_pool.append(storagePool)
+                zone_no += 1
+
+        self.vm_zwps = VirtualMachine.create(
+            self.apiclient,
+            self.testdata["small"],
+            templateid=self.template.id,
+            accountid=self.account.name,
+            domainid=self.account.domainid,
+            serviceofferingid=self.service_offering_zwps.id,
+            diskofferingid=self.disk_offering_zwps.id,
+            zoneid=self.zone.id,
+        )
+
+        self.cleanup.append(self.vm_zwps)
+
+        # Step 1
+        volumes_root_list = list_volumes(
+            self.apiclient,
+            virtualmachineid=self.vm_zwps.id,
+            type=ROOT,
+            listall=True
+        )
+
+        root_volume = volumes_root_list[0]
+
+        if root_volume.storage != self.zone_pool[0].name:
+            self.skipTest("Root Volume not in Zone-Wide Storage Pool !")
```

When we are creating vm with zone wide primary storage(service offering used has zone wide primary storage tag) root volume should be created in zwps as per the tagging. So instead of skipping the test, fail the test if the root volume is not in zwps.